parlai/core/image_featurizers.py

```diff
@@ -109,7 +131,8 @@ def extract(self, image, path=None):
         transform = self.transform(image).unsqueeze(0)
         if self.use_cuda:
             transform = transform.cuda()
-        feature = self.netCNN(transform)
+        with self.torch.no_grad():
+            feature = self.netCNN(transform)
```

Comment:
why the nograd? MAYBE I WANNA FINETUNE MY 900M PARAMETER MODEL, HUH KURT?

Reply:
(real question though.)

Reply:
Speeds up computation; also recommended [here](https://pytorch.org/hub/facebookresearch_WSL-Images_resnext/) (see third code block)